WalletWasabi.Fluent/ViewModels/Wallets/Send/TransactionPreviewViewModel.cs

```diff
@@ -222,6 +229,19 @@ private async Task OnChangePocketsAsync()
 		}
 	}
 
+	private async Task OnChangeCoinsAsync()
+	{
+		var selectCoinsDialog =
+			await NavigateDialogAsync(new SelectCoinsDialogViewModel(_walletViewModel));
+
+		if (selectCoinsDialog.Kind == DialogResultKind.Normal && selectCoinsDialog.Result is { })
+		{
+			_info.Coins = selectCoinsDialog.Result;
+			_info.ChangelessCoins = Enumerable.Empty<SmartCoin>(); // Clear ChangelessCoins on pocket change, so we calculate the suggestions with the new coins.
+			await BuildAndUpdateAsync(BuildTransactionReason.PocketChanged); // TODO: check if PocketChanged reason is good for this case too (maybe rename)
```

Comment:
// TODO: 
Will this be done in this PR ?

Reply:
cc @soosr

Reply:
TODO was added by me, and it can be removed. I checked it and that is correct.